crates/noirc_evaluator/src/ssa/acir_gen.rs

```diff
@@ -71,6 +71,12 @@ impl InternalVar {
         }
         evaluator.create_intermediate_variable(expr)
     }
+
+    fn eq(&self, other: &Self) -> bool {
```

Comment:
Is there a reason this is not in an `impl Eq for InternalVar`?

Reply:
No, I will do it to avoid another TODO!

Reply:
Done